utilities/cassandra/format.cc

```diff
@@ -153,7 +161,16 @@ void Tombstone::Serialize(std::string* dest) const {
   rocksdb::cassandra::Serialize<int64_t>(marked_for_delete_at_, dest);
 }
 
-std::unique_ptr<Tombstone> Tombstone::Deserialize(const char *src,
+bool Tombstone::Collectable() const {
+  // todo: we can only collect a tombstone if
+  // * marked_for_delete_at_ + gc_grace_seconds < now
+  // * and there are no live data for the key in higher level.
+  // We cannot get info for the second condidtion yet, so we
+  // will just keep tombstone around for now.
+  return false;
```

Comment:
Yeah. But we still do not have information on whether there are live data on the higher level sstable files yet. It feels weird passing in gc_grace_seconds but has no place using it.